.changeset/violet-dots-relate.md

```diff
@@ -0,0 +1,30 @@
+---
+'@backstage/core-components': patch
+---
+
+Added option to pass additional headers to `<ProxiedSignInPage />`
```

Comment:
I think it's enough to add ", which are passed along with the request to the underlying provider" or similar here, and skip the rest of the text. Since this ends up in a changelog and not referred to as documentation etc, it's OK to skip explaining what the component is and how to use it - that should instead go in docs or in the doc-comment of the class itself where it can be reached long-term by users.

Reply:
I'm the culprit here 😅 - thought of giving more context to the readers. But yeah, we can move over the text to the docs and say "Check out https://backstage.io/docs/auth/#sign-in-with-proxy-providers for more details."

Reply:
I've moved the explanations to the docs with more examples